yarn-project/simulator/src/avm/serialization/instruction_serialization.ts

```diff
@@ -84,8 +84,11 @@ export enum Opcode {
   MSM,
   // Conversion
   TORADIXBE,
+  // Adapt MAX_OPCODE_VALUE below if TORADIXBE is not the last opcode anymore
 }
 
+export const MAX_OPCODE_VALUE: number = Opcode.TORADIXBE;
```

Comment:
Hmm I guess you couldn't add a sentinel `NUM_OPCODES` because the `Record`s oblige you to define gas costs for it/etc?

otherwise you can use something like this:

```
enum MyEnum {
    ZoomIn = 600,
    ZoomOut,
    Close, 
    Open,
    // ....
}

const MAX_OPCODE_VALUE = Math.max(...Object.values(MyEnum).map(k => +k).filter(k => !isNaN(k)));
```

Reply:
Exactly. Sentinel did not make the job because of this. I will try your suggestion.